pkg/build/api/types.go

```diff
@@ -177,6 +190,9 @@ const (
 	// StatusReasonExceededRetryTimeout is an error condition when the build has
 	// not completed and retrying the build times out.
 	StatusReasonExceededRetryTimeout = "ExceededRetryTimeout"
+
+	// StatusReasonSkipped indicates that the build was skipped.
+	StatusReasonSkipped = "Skipped"
```

Comment:
The reason would not be the same as the condition - the reason would be *why*, like "BuildRunning" or "WaitingForBuild"